### PR TITLE
Please review lists, typography and navbar!

### PR DESCRIPTION
**Pull request by muskanbagrecha**

### CODE REVIEW

File: navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        z-index: 1;
+        transition: all 0.3s ease-in-out;
+    }
+
+    /*
+    .search-form--active{
+        left: 50%;
+        transform: translate(-50%, 0%);
+    } */
+}
```
Comment (anshulraheja) [inline navigation/navigation.css]: remove comments

File: navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,206 @@
+.navigation{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    box-sizing: border-box;
+    background-color: var(--nav-color);
+    /* position: fixed; */
+    width: 100%;
+    padding: 1rem;
```
Comment (anshulraheja) [inline navigation/navigation.css]: I just checked, this class is being used in the documentation navbar and example navbar so the height of 5rem is causing an issue, you can remove that

**Review by anshulraheja — APPROVED** [review]

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 🚀

File: navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+.navigation .nav-list li a:hover, .nav-list .active{
+    color: var(--color-dark);
+    border-bottom: 2px solid var(--color-dark);
+    transition: all 0.2s ease-in-out;
+}
+
+.navigation li a, .navigation li a:visited{
+    text-decoration: none;
+    color: var(--color-complementary);
```
Comment (PuneetP16) [inline navigation/navigation.css:42]: This property is not applying, check for specificity issues.

**Review by PuneetP16 — APPROVED** [review]
Good implementation of Search input box. Looks good to merge.

**Pull request merged**